x-pack/legacy/plugins/monitoring/public/views/base_controller.js

```diff
@@ -152,9 +168,11 @@ export class MonitoringViewBaseController {
     this.onBrush = ({ xaxis }) => {
       removePopstateHandler();
       const { to, from } = xaxis;
+      const timezone = config.get('dateFormat:tz');
+      const offset = getOffsetInMS(timezone);
```

Comment:
More of nitpick, but I think `timezone` and `offset` can be calculated just once in the constructor, unless `config.get` always gets the latest/updated value